release-tools/prow.sh

```diff
@@ -652,103 +515,74 @@ find_deployment () {
 # image versions with that canary version.
-install_csi_driver () {
-    local images deploy_driver
+install_cosi_driver () {
+#    local images deploy_driver
     images="$*"
-
-    if [ "${CSI_PROW_DEPLOYMENT}" = "none" ]; then
-        return 1
-    fi
-
-    if ${CSI_PROW_BUILD_JOB}; then
-        # Ignore: Double quote to prevent globbing and word splitting.
-        # Ignore: To read lines rather than words, pipe/redirect to a 'while read' loop.
-        # shellcheck disable=SC2086 disable=SC2013
-        for i in $(grep '^\s*CMDS\s*=' Makefile | sed -e 's/\s*CMDS\s*=//'); do
-            kind load docker-image --name csi-prow $i:csiprow || die "could not load the $i:latest image into the kind cluster"
-        done
-    fi
-
-    if deploy_driver="$(find_deployment "$(pwd)/deploy")"; then
-        :
-    elif [ "${CSI_PROW_DRIVER_REPO}" = "none" ]; then
-        return 1
-    else
-        git_checkout "${CSI_PROW_DRIVER_REPO}" "${CSI_PROW_WORK}/csi-driver" "${CSI_PROW_DRIVER_VERSION}" --depth=1 || die "checking out CSI driver repo failed"
-        if deploy_driver="$(find_deployment "${CSI_PROW_WORK}/csi-driver/deploy")"; then
-            :
-        else
-            die "deploy.sh not found in ${CSI_PROW_DRIVER_REPO} ${CSI_PROW_DRIVER_VERSION}. To disable E2E testing, set CSI_PROW_DRIVER_REPO=none"
-        fi
-    fi
-
-    if [ "${CSI_PROW_DRIVER_CANARY}" != "stable" ]; then
-        images="$images IMAGE_TAG=${CSI_PROW_DRIVER_CANARY}"
-    fi
-    # Ignore: Double quote to prevent globbing and word splitting.
-    # It's intentional here for $images.
-    # shellcheck disable=SC2086
-    if ! run env "CSI_PROW_TEST_DRIVER=${CSI_PROW_WORK}/test-driver.yaml" $images "${deploy_driver}"; then
-        # Collect information about failed deployment before failing.
-        collect_cluster_info
-        (start_loggers >/dev/null; wait)
-        info "For container output see job artifacts."
-        die "deploying the CSI driver with ${deploy_driver} failed"
-    fi
 }
 
-# Installs all nessesary snapshotter CRDs  
-install_snapshot_crds() {
-  # Wait until volumesnapshot CRDs are in place.
-  CRD_BASE_DIR="https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/${CSI_SNAPSHOTTER_VERSION}/config/crd"
-  kubectl apply -f "${CRD_BASE_DIR}/snapshot.storage.k8s.io_volumesnapshotclasses.yaml" --validate=false
-  kubectl apply -f "${CRD_BASE_DIR}/snapshot.storage.k8s.io_volumesnapshots.yaml" --validate=false
-  kubectl apply -f "${CRD_BASE_DIR}/snapshot.storage.k8s.io_volumesnapshotcontents.yaml" --validate=false
+# Installs all nessesary CRDs  
+install_crds() {
+  # Wait until cosi CRDs are in place.
+#  CRD_BASE_DIR="https://github.com/kubernetes-sigs/container-object-storage-interface-api/crds"
+  CRD_BASE_DIR="https://raw.githubusercontent.com/brahmaroutu/api/proper_crds/crds"
+  kubectl apply -f "${CRD_BASE_DIR}/objectstorage.k8s.io_bucketclasses.yaml" --validate=false
```

Comment:
The install location should be dynamic. If this is testing a PR at the API repo "https://github.com/kubernetes-sigs/container-object-storage-interface-api", it should read from the  API repo.  In other cases, it should either install from master or a stable branch of the API repo.

See this PR for example: https://github.com/kubernetes-csi/csi-release-tools/pull/113

Reply:
Fixed